README.md

```diff
@@ -95,6 +95,29 @@ fpath+=~/.zfunc
 ```
 
 
+## Docker example
+
+```dockerfile
+FROM python:3.7-alpine
+
+MAINTAINER name <email>
```

Comment:
[MAINTAINER is deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

You should use `LABEL maintainer="name <email>"` instead, and also put it at the end to avoid cache invalidation when you only change the label :slightly_smiling_face:

Reply:
nice to know, will update!